README.md

```diff
@@ -170,7 +192,9 @@ If you would like to modify or sanitize the HTML before rendering it, then `Html
 
 ### Getters:
 
-Currently the only getter is `Html.tags`. This provides a list of all the tags the package renders. The main use case is to assist in blacklisting elements using `tagsList`. See an [example](#example-usage---tagslist---excluding-tags) below.
+1. `Html.tags`. This provides a list of all the tags the package renders. The main use case is to assist in blacklisting elements using `tagsList`. See an [example](#example-usage---tagslist---excluding-tags) below.
```

Comment:
Please don;t use the term blacklisting, but use 'excluding' instead.

Reply:
Fixed.